base/Dockerfile

```diff
@@ -754,6 +754,16 @@ ARG START_RSYSLOGD="/usr/sbin/rsyslogd -n"
 RUN \
     echo "${NB_USER} ALL=(ALL) NOPASSWD: ${START_CRON}, ${START_NETDATA}, ${START_RSYSLOGD}" >> /etc/sudoers.d/${NB_USER}
 
+# Install the French Locale
+RUN \
```

Comment:
I suggest to add a small comment on why we're using fr_FR instead of fr_CA

Reply:
Only browser needs to be in fr_FR. Installed locale can be fr_CA